src/shapes/FocusedInputShape.js

```diff
@@ -5,4 +5,10 @@ import {
   END_DATE,
 } from '../../constants';
 
-export default PropTypes.oneOf([START_DATE, END_DATE]);
+export default function(props, propName, componentName){
+  const item = props[propName];
+  if (item === START_DATE || item === END_DATE || item === ''){
+    return null;
+  }
+  return new Error(`${propName} must be an empty string, "${START_DATE}", or "${END_DATE}"`);
```

Comment:
`null` seems to be the case when a prop is either omitted or explicitly passed null. In my case, I omitted it and was not warned, so I had some confusion when first using this lib. This PR was my attempt to rectify that situation